Dockerfile

```diff
@@ -1,8 +1,23 @@
-FROM alpine
-MAINTAINER Brian O'Kelley <bokelley@appnexus.com>
-ADD prebid-server prebid-server
-COPY static static/
-COPY stored_requests/data stored_requests/data
+FROM golang:latest
```

Comment:
Are we sure we want to use Stretch instead of Alpine? We use Alpine for all of the other images. If we want to stay with Alpine that tag would be `golang:alpine`

Reply:
sounds good - i'll update